roles/observe_chargeback/README.md

```diff
@@ -0,0 +1,59 @@
+Role Name
+=========
+The **`observe_chargeback`** role is designed to test the **RHOSO Cloudkitty** feature. These tests are specific to the Cloudkitty feature. Tests that are not specific to this feature (e.g., standard OpenStack deployment validation, basic networking) should be added to a common role.
+
+Requirements
+------------
+The following resources are required in the OpenStack cloud
+* network called public
+* network called private
+* security group called basic with ssh and tcp enabled to the VMs
+
+
+It relies on the following being available on the target or control host:
+
+* This role requires **Ansible 2.9** or newer.
+* The **OpenStack CLI client** must be installed and configured with administrative credentials (e.g., source the `openrc` file).
+* Required Python libraries for the `openstack` CLI (e.g., `python3-openstackclient`).
+* Connectivity to the OpenStack API endpoint.
+
+Role Variables
+--------------
+The role uses a few primary variables to control the testing environment and execution.
+
+| Variable | Default Value | Description |
+| `openstack_cmd` | `openstack` | The command used to execute OpenStack CLI calls. This can be customized if the binary is not in the standard PATH. |
```

Comment:
```suggestion
| Variable | Default Value | Description |
|----------|---------------|-------------|
| `openstack_cmd` | `openstack` | The command used to execute OpenStack CLI calls. This can be customized if the binary is not in the standard PATH. |
```